assets/_scss/components/_accordions.scss

```diff
@@ -1,51 +1,53 @@
 #accordion-menu {
-  border: 1px solid $color-grey-600;
-  border-radius: 5px;
-  color: $color-grey-900;
-  margin: 0;
-  overflow: hidden;
-  padding: 0;
-
-  li {
-    background-color: $color-grey-200;
-    border-bottom: 1px solid $color-grey-600;
-    font-family: $font-sans;
-    list-style: none;
+  ul {
```

Comment:
@msecret good question! I moved the `code-sample` class to be more specific to the area in which the code snippets live, rather than wrapping the entire area. It made more sense to separate those out.